lib/dynamics_crm/xml/attributes.rb

```diff
@@ -33,6 +33,8 @@ def get_type(key, value)
             type = "FetchExpression"
           when Money
             type = "Money"
+          when DynamicsCRM::Metadata::Double
```

Comment:
Ruby doesn't have a double type. BigDecimal/Float are mapped to decimal in CRM already, so this seemed the way to go.